packages/drivers/odsp-urlResolver/package.json

```diff
@@ -107,7 +109,6 @@
 		"@fluidframework/build-tools": "^0.29.0",
 		"@fluidframework/eslint-config-fluid": "^3.3.0",
 		"@fluidframework/mocha-test-setup": "workspace:~",
-		"@fluidframework/odsp-urlresolver-previous": "npm:@fluidframework/odsp-urlresolver@2.0.0-internal.8.0.0",
```

Comment:
Interesting... do we know why this was using a `npm:` dependency instead of a `workspace:` one?

Reply:
This is the pattern used for installing "previous" versions of the same package for use in the type-tests.

Is there a reason we're removing it? We still need this in order for the package's type-tests to function.

Reply:
Seems like those previous packages are generated by type-tests. It got removed somehow along with the package re-built. Running type tests again...

Reply:
The previous packages are just the version of the package we specify inline, as published to npm. These dependencies are _used_ by the type-tests, but they aren't created by them.

Reply:
I see. Adding type test file back